tket/src/OpType/include/OpType/EdgeType.hpp

```diff
@@ -42,14 +42,21 @@ enum class EdgeType {
    * port of one op to a classical input port of another, not corresponding to
    * any allocated @ref Bit.
    */
-  Boolean
+  Boolean,
+
+  /**
+   * A wire to connect the wasm ops in the order they should be executed in.
+   * Corresponding to some allocated @ref WASMUID.
+   */
+  WASM
```

Comment:
Have you checked that the https://cqcl.github.io/tket/pytket/api/utils.html#pytket.utils.Graph code is compatible with the new edge type?

Reply:
Yes I have checked that, it is now working as expected. I fixed the issues we found last year with the wrong conected bool wires as well. (not sure if we had a github issues for that, I have not found one)